website/client/js/services/userServices.js

```diff
@@ -151,7 +151,7 @@ angular.module('habitrpg')
         $http({
           method: method,
           url: url + queryString,
-          body: body,
```

Comment:
I noticed that it needed to be `data` instead of `body` when looking up [$http](https://docs.angularjs.org/api/ng/service/$http#usage) on AngularJS